packages/manager/modules/pci/src/projects/project/storages/cold-archive/cold-archive.routing.js

```diff
@@ -24,7 +25,26 @@ export default /* @ngInject */ ($stateProvider) => {
               },
         ),
     resolve: {
-      guides: () => GUIDES,
+      guides: /* @ngInject */ (coreConfig, $translate) =>
+        reduce(
+          GUIDES,
```

Comment:
I think we could do without importing `reduce` here
```suggestion
        GUIDES.reduce(
```

Reply:
Done.